app/components/conversations/conversation-part-comment.js

```diff
@@ -4,7 +4,7 @@ import { alias } from '@ember/object/computed';
 import { inject as service } from '@ember/service';
 
 export default Component.extend({
-  classNames: ['conversation-part'],
+  classNames: ['conversation-part, conversation-part-comment'],
```

Comment:
I'm not sure precisely where your tests are breaking, but this is certainly a syntactical problem.